assets/css/app.scss

```diff
@@ -1,3 +1,28 @@
+// this is to lock the footer on the bottom without having to deal with position
+html, body {
+    height: 100%;
+
+    // this is the final part to lock the fotter to the bottom you are welcome to try and figure out the best dynamic way to set this but have fun with that function
+    #app {
+        @media(min-height: 558px){
+            min-height: 51.6073%;
+        }
+
+        @media(min-height: 675px){
+            min-height: 59.9953%;
+        }
+
+        @media(min-height: 937px){
+            min-height: 71.1812%;
+        }
+
+        @media(min-height: 1297px){
+            min-height: 79.18%;
+        }
+    }
+}
+
```

Comment:
This doesn't seem right, why do we need this?

Reply:
This is to force the footer to always be at the bottom of the screen view without having to deal with javascript to set it. And this is something that should always be done with the footer to force it on the bottom of the screen and not have it be at top half of the view